### PR TITLE
Made grunt to log each file only on verbose

### PR DESCRIPTION
**Pull request by ascripcaru**
@vladikoff sorry for the mess

### CODE REVIEW

**Comment by ascripcaru** [comment]
@vladikoff sorry for the mess

File: tasks/lib/compress.js [hunk 1]
```diff
@@ -34,16 +34,22 @@ module.exports = function(grunt) {
   // 1 to 1 gziping of files
   exports.gzip = function(files, done) {
     exports.singleFile(files, zlib.createGzip, 'gz', done);
+    grunt.log.writeln('Compressed ' + chalk.cyan(files.length)
+      + grunt.util.pluralize(files.length, ' file/ files'));
```
Comment (XhmikosR) [inline tasks/lib/compress.js]: Is the space really needed after the forward slash?

**Comment by vladikoff** [comment]
Also I think we use .log.ok (see other plugins), for the final message, to get a nice green arrow

File: tasks/lib/compress.js (hunk 1 above)
Comment (ascripcaru) [inline tasks/lib/compress.js]: Yeah to add space between the number and the word

**Comment by ascripcaru** [comment]
@vladikoff @XhmikosR updated

**Comment by vladikoff** [comment]
Thank you so much!

**Comment by ascripcaru** [comment]
cheers. Waiting for the new release

**Pull request merged**